src/main/java/com/reshiftsecurity/education/ReshiftEducationService.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 SpotBugs plugin contributors
+ *
+ * This file is part of IntelliJ SpotBugs plugin.
+ *
+ * IntelliJ SpotBugs plugin is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * IntelliJ SpotBugs plugin is distributed in the hope that it will
+ * be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with IntelliJ SpotBugs plugin.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.reshiftsecurity.education;
+
+import org.apache.commons.lang.StringUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReshiftEducationService {
+    public static ReshiftVulnerabilityDetails getVulnerabilityDetails(String vulnerabilityType) {
+        List<ReshiftDevContent> devContent = new ArrayList<>();
+        String rawHtml = getRawVulnerabilityDetails(vulnerabilityType);
+        String[] sections = rawHtml.split("<h1>\\|");
```

Comment:
just another note here, doesn't jsoup have facilities for parsing HTML as well as section navigation?

Reply:
Yes it does, the only issue is I noticed some of the generated HTML had multiple h1 tags. So I took the lazy route. But I believe once this is switched to JSON we won't need this. I'll make it a priority to do so.

Reply:
Yes, however if we are switching to JSON I believe this code will go away: https://softwaresecured.atlassian.net/browse/OM-1366